File: 1950-sign-of-the-product-of-an-array/1950-sign-of-the-product-of-an-array.py
class Solution(object):
    def arraySign(self, nums):
        pro = 1
        for i in range(len(nums)):
            pro *=nums[i]
            
        if pro > 0:
            return 1
        elif pro < 0:
            return -1
        else:
            return 0
        

File: 1950-sign-of-the-product-of-an-array/1950-sign-of-the-product-of-an-array.java
class Solution {
    public int arraySign(int[] nums) {
        short pro = 1;
        for(int i =0; i < nums.length; i++){
            pro *= nums[i];
            if(pro == 0){
                return 0;
            }else if(pro > 0){
                pro = 1;
            }else{
                pro = -1;
            }
        }

        return pro;
        
    }
}